cypress/integration/storefront/navigation/viewport.spec.js

```diff
@@ -0,0 +1,33 @@
+describe('Nav Menus', function () {
+
+  beforeEach(() => {
+    cy.visit('');
```

Comment:
One little thing I have to change in my own storefront tests as well:
`cy.visit('/');` instead of using an empty string.

Does the same, while looking better.

Reply:
Fixed in 4650cace9e41bd1b5ca926ae146d57fb33ccf6df